build/tools/templates/SharedFxSymbols/Microsoft.AspNetCore.All.symbols.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.AspNetCore.All.symbols</id>
+    <version>$VERSION$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
+    <projectUrl>http://www.asp.net/</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <description>Symbol packages for Microsoft.AspNetCore.All shared framework</description>
+    <copyright>© .NET Foundation. All rights reserved.</copyright>
+    <tags>aspnetcore</tags>
+    <serviceable>true</serviceable>
+    <repository type="git" url="https://github.com/aspnet/Universe" />
+  </metadata>
+  <files>
+    <file src="*.pdb" target="" />
```

Comment:
Should these be nested according to RID? Are there symbols for multiple RIDs in the package?